petrinet/examples/flux_typed.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "Two City Stratification Model",
+  "schema": "https://raw.githubusercontent.com/DARPA-ASKEM/Model-Representations/petrinet_v0.5/petrinet/petrinet_schema.json",
+  "description": "Stratification spatially between two cities model created by Micah. Modified by Patrick.",
+  "model_version": "0.1",
+  "model": {
+    "states": [
+      {
+        "id": "Rgn_1",
+        "name": "Region 1",
+        "description": "Number of individuals physically located in region 1 relative to the total population"
+      },
+      {
+        "id": "Rgn_2",
+        "name": "Region 2",
+        "description": "Number of individuals physically located in region 2 relative to the total population"
+      }
+    ],
+    "transitions": [
+      {
+        "id": "travel_1_2",
+        "input": ["Rgn_1"],
+        "output": ["Rgn_2"],
+        "properties": {
+          "name": "Traveling 1 -> 2",
+          "description": "Individuals traveling from region 1 to region 2"
+        }
+      },
+      {
+        "id": "travel_2_1",
+        "input": ["Rgn_2"],
+        "output": ["Rgn_1"],
+        "properties": {
+          "name": "Traveling 2 -> 1",
+          "description": "Individuals traveling from region 2 to region 1"
+        }
+      }
+    ]
+  },
+  "semantics": {
+    "ode": {},
+    "typing": {
+      "system": {
```

Comment:
I think if possible we should have `system` contain only `states` and `transitions` to be consistent with the structure of existing models we have in tds.

Reply:
Nevermind, this is a non issue!